src/server/handlers/launchpad.js

```diff
@@ -340,6 +340,47 @@ export const internalFindSnap = async (repositoryUrl) => {
   });
 };
 
+const internalFindSnapsByPrefix = (urlPrefix) => {
+  const username = conf.get('LP_API_USERNAME');
+
+  return getLaunchpad().named_get('/+snaps', 'findByURLPrefix', {
+    parameters: {
+      url_prefix: urlPrefix,
+      owner: `/~${username}`
+    }
+  })
+  .then(result => {
+    return result.entries;
+  })
+  .catch((error) => {
+    // At least for the moment, we just wrap the error we get from
+    // Launchpad.
+    return error.response.text().then((text) => {
+      throw new PreparedError(error.response.status, {
+        status: 'error',
+        payload: {
+          code: 'lp-error',
+          message: text
+        }
+      });
+    });
+  });
+};
+
+export const findSnaps = (req, res) => {
+  internalFindSnapsByPrefix(`https://github.com/${req.query.owner}/`)
+    .then((snaps) => {
+      return res.status(200).send({
+        status: 'success',
+        payload: {
+          code: 'snaps-found',
```

Comment:
LP returns success with empty list when nothing is found.

I keep it like this, because I'm not sure if we need 404 here in case of empty list.